### PR TITLE
Milansprint4

### PR DESCRIPTION
**Pull request by milanopatela**
Added linked account functionality

### CODE REVIEW

File: BaseProject#1/SOEN341InstagramReplica/SOEN341InstagramReplica/Views/LinkedAccounts/Delete.cshtml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        </dd>
+
+    </dl>
+
+    @using (Html.BeginForm()) {
+        @Html.AntiForgeryToken()
+
+        <div class="form-actions no-color">
+            <input type="submit" value="Delete" class="btn btn-default" /> |
+            @Html.ActionLink("Back to List", "Index")
```
Comment (zaad97) [inline BaseProject#1/SOEN341InstagramReplica/SOEN341InstagramReplica/Views/LinkedAccounts/Delete.cshtml]: localhost/LinkedAccounts/Index does not exist.
Try to add the Home controller
Something like this: @Html.ActionLink("Back to List", "Index","Home")

File: BaseProject#1/SOEN341InstagramReplica/SOEN341InstagramReplica/Controllers/UsersController.cs [hunk 2]
```diff
@@ -270,5 +304,85 @@ public ActionResult FollowOrUnfollowUser(int newFollowStatus, int loggedInUser,
             return Json(new { user, Status = "Ok", Error = "" });
         }
 
+        public ActionResult LinkAccount()
+        {
+            return View();
+        }
+
+        [HttpPost]
+        public ActionResult LinkAccount(User model)
```
Comment (zaad97) [inline BaseProject#1/SOEN341InstagramReplica/SOEN341InstagramReplica/Controllers/UsersController.cs:313]: The user is not supposed to be able to link the same account.
For example, if I am using my account "bob1", then i cannot link with "bob1". It must be another account.

File: BaseProject#1/SOEN341InstagramReplica/SOEN341InstagramReplica/Views/Users/RemoveLink.cshtml [hunk 3]
```diff
@@ -0,0 +1,40 @@
+﻿@model SOEN341InstagramReplica.Models.LinkedAccount
```
Comment (zaad97) [inline BaseProject#1/SOEN341InstagramReplica/SOEN341InstagramReplica/Views/Users/RemoveLink.cshtml]: How am I suppose to go to this page?
Also what is the difference between ~/Views/Users/RemoveLink.cshtml and ~/Views/LinkedAccounts/Delete.cshtml ?

**Review by zaad97 — CHANGES_REQUESTED** [review]
Check the review before merging

File: BaseProject#1/SOEN341InstagramReplica/SOEN341InstagramReplica/Views/LinkedAccounts/Delete.cshtml (hunk 1 above)
Comment (milanopatela) [reply BaseProject#1/SOEN341InstagramReplica/SOEN341InstagramReplica/Views/LinkedAccounts/Delete.cshtml]: understood

File: BaseProject#1/SOEN341InstagramReplica/SOEN341InstagramReplica/Controllers/UsersController.cs (hunk 2 above)
Comment (milanopatela) [reply BaseProject#1/SOEN341InstagramReplica/SOEN341InstagramReplica/Controllers/UsersController.cs:313]: your right, will fix

File: BaseProject#1/SOEN341InstagramReplica/SOEN341InstagramReplica/Views/Users/RemoveLink.cshtml (hunk 3 above)
Comment (milanopatela) [reply BaseProject#1/SOEN341InstagramReplica/SOEN341InstagramReplica/Views/Users/RemoveLink.cshtml]: will deal with

Comment (milanopatela) [reply BaseProject#1/SOEN341InstagramReplica/SOEN341InstagramReplica/Views/Users/RemoveLink.cshtml]: removed removelink.cshhtml

**Pull request merged**